timezone/src/clients/timezone_proposal.rb

```diff
@@ -28,15 +26,7 @@ def main
         @force_reset = Ops.get_boolean(@param, "force_reset", false)
         @language_changed = Ops.get_boolean(@param, "language_changed", false)
 
-        @m = Convert.to_map(
-          SCR.Execute(path(".target.bash_output"), "/bin/date +%Y")
-        )
-        Builtins.y2milestone("date call: %1", @m)
-
-        if Ops.less_than(
-            Builtins.tointeger(Ops.get_string(@m, "stdout", "0")),
-            2004
-          )
+        if Time.now.year < 2015
```

Comment:
It's already better than before, but in the future we should probably do something more... well... elegant (like using the creation/modification time of some file) in order to decide if we are in the past.